plugin.yml

```diff
@@ -22,7 +22,4 @@ configuration:
   required:
     - server
     - auth
-  additionalProperties: false
-  dependencies:
-    auth: [ method ]
```

Comment:
why these changes?

Reply:
Fixed this in a more recent commit, we want `auth` to be a required property.